tests/test/test_admin_checkout_no_user_not_found_error.py

```
@@ -0,0 +1,64 @@
+
+    # 🔍 SOFT CHECK (non-blocking UI validation)
+    success_visible = False
+
+    try:
+        checkout_page.get_wait(5).until(
+            lambda d: len(d.find_elements(
+                By.XPATH, "//*[contains(text(),'checked out')]"
+            )) > 0
+        )
+        success_visible = True
+    except:
```

Ruff (0.15.9)

[error] Do not use bare `except`

(E722)